### ISSUE TITLE
A11Y weekly check in - 31 October 2023

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#48]
Smera and I spent most of the call discussing the focus/hover/current styles for the header nav links.

Some things we discussed:

- Gabriel's concerned with the current design iteration not being maintainable (too many color combinations)
- Would it be helpful to start with designs in monochrome? Do we have too much color right now?
- What if we collapsed/merged the hover and current styles like the header nav on a11yproject.com does?

Decisions made:

- For now, we will keep brainstorming the header nav. We may wait for Tania to come back before making a final decision. Gabriel may implement an alternative design if it's not too hard to implement.
- Border radius inconsistencies (apparent on focus visible):
  - Header nav: all items should have same border-radius = 2px, except search bar
  - Sidebar nav: should follow Figma (also 2px)
  - Regular links: no border radius

Action items:

- [ ] Smera to implement a Figma frame of hover nav styles in which the hover and current styles are collapsed (or differ only in text color)
- [ ] Smera to potentially ask in tomorrow's team call for guidance on what is needed to make the header nav (1) conform to WCAG and (2) be a good user experience for abled and disabled users.
- [ ] Gabriel to finish updating the header nav to match the Figma board
- [ ] Gabriel to work on the sidebar dropdowns
